### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by seanyboy49**

### CODE REVIEW

File: src/components/header/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const StyledLink = styled(Link)`
+  color: black;
+  text-decoration: none;
+  &:hover {
+    color: grey;
+  }
+`
+
+const Header = () => {
+  console.log("header")
```
Comment (JinaZhu) [inline src/components/header/index.js]: remove console log

File: src/components/header/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import { Link } from "gatsby"
+
+import logoSmall from "../../images/logo-small.svg"
+
+const NavBar = styled.nav`
```
Comment (JinaZhu) [inline src/components/header/index.js]: create `style.js` for styled-components

File: src/pages/papaya.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from "react"
```
Comment (JinaZhu) [inline src/pages/papaya.js]: Let's remove this page

**Review by JinaZhu — CHANGES_REQUESTED** [review]

File: src/components/header/index.js (hunk 2 above)
Comment (seanyboy49) [reply src/components/header/index.js]: Ok good call! <3

**Review by JinaZhu — APPROVED** [review]
Good job <3

**Pull request merged**